src/common/options.cc

```diff
@@ -897,6 +897,36 @@ std::vector<Option> get_global_options() {
     .set_default(true)
     .set_description(""),
 
+    /* -- mon: osdmap prune (begin) -- */
+    Option("mon_osdmap_full_prune_enabled", Option::TYPE_BOOL, Option::LEVEL_ADVANCED)
+    .set_default(false)
+    .set_description("Enables pruning full osdmap versions when we go over a given number of maps")
+    .add_see_also("mon_osdmap_full_prune_min")
+    .add_see_also("mon_osdmap_full_prune_interval")
+    .add_see_also("mon_osdmap_full_prune_txsize"),
+
+    Option("mon_osdmap_full_prune_min", Option::TYPE_UINT, Option::LEVEL_ADVANCED)
+    .set_default(10000)
+    .set_description("Minimum number of versions in the store to trigger full map pruning")
+    .add_see_also("mon_osdmap_full_prune_enabled")
```

Comment:
nit, i think we can assume the yet-written tool will take `Option("foo").add_see_also("bar")` as a link from "foo" to "bar", and hence "bar" will be linked to "foo" also.

Reply:
I do not know what tool is this you are talking about, but -- even if it is in the works -- it seems that assumption to be weird. I understand the `add_see_also()` as a unidirectional link, to further instruct the user to know more about a given topic; sometimes we'll have bidirectional links, but not necessarily.
In any case, I presume this tool will be smart enough to either take it as is, or complain about it and having us remove one of the links. When the time comes, I'll be all for fixing this to follow said tool's semantics.

Reply:
okay. i think it will be bidirectional. but, agreed. we can fix it when the tool is around. BTW, for the "tool", see https://trello.com/c/wBc0R9HV/6-config-options-docs and http://pad.ceph.com/p/config-annotation

> script to generate .rst docs for docs.ceph.com